### PR TITLE
Refactor into smaller components

### PR DESCRIPTION
**Pull request by Charliekenney23**
Thanks for starting the project! ❤️ 

- Refactored into smaller components with scoped styles for maintainability.
- Adopted prettier and eslint as code style was inconsistent (let me know how you want me to configure it, just used the standard for Vue projects).
- Added Github workflow for smoke testing.

### CODE REVIEW

File: covid-19-support/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Charliekenney23) [inline covid-19-support/.prettierrc]: I would love input on how this should be configured.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: CI
```
Comment (Charliekenney23) [inline .github/workflows/ci.yml]: This workflow is to ensure that the code passes the linter and builds successfully. A simple smoke test

File: covid-19-support/src/components/Highlights.vue [hunk 3]
```diff
@@ -1,51 +1,167 @@
...
-        </div>
-        <div class="col-6 col-md-3 order-md-4">                    
-            <value-box :icon="valueBoxes[2].icon" :title="valueBoxes[2].title" :value="valueBoxes[2].value" class="bg-green"/> 
-        </div>
-        <div class="col-6 col-md-3 order-md-3">              
-            <value-box :icon="valueBoxes[3].icon" :title="valueBoxes[3].title" :value="valueBoxes[3].value" class="bg-blue"/>
-        </div>
+<div class="row highlights">
+    <div 
+        class="col-6 col-md-3 order-md-1"
```
Comment (readingdancer) [inline covid-19-support/src/components/Highlights.vue]: This line unfortunately breaks the functionality on mobile, if you look above at the original code you will see the classes are different. There might be a nicer way to write what I did, I had wanted to use a v-for, but at the time I wasn't sure how best to handle those different classes, hence just wrote them out in full :)

**Comment by Charliekenney23** [comment]
Rebased onto master and addressed that bit of feedback

**Comment by readingdancer** [comment]
@all-contributors please add charliekenney23 to code

**Pull request merged**